src/ng/directive/ngModel.js

```diff
@@ -1110,14 +1113,14 @@ var DEFAULT_REGEXP = /(\s+|^)default(\s+|$)/;
     <file name="index.html">
       <div ng-controller="ExampleController">
         <form name="userForm">
-          Name:
-          <input type="text" name="userName"
+          <label for="userName">Name:</label>
+          <input type="text" name="userName" id="userName"
```

Comment:
Seems like it'd be more consistent with the document to put the `id` attribute first.